samples/valueentity-counter/package.json

```diff
@@ -7,11 +7,7 @@
     "npm": ">=6.0.0"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.3.0",
```

Comment:
Note for the reviewers, this project is built and tested in CI, so we are confident that the dependencies are not needed.